web-ui/src/main/resources/catalog/components/userfeedback/partials/userfeedback.html

```diff
@@ -1,43 +1,57 @@
-<div>
-  <section class="gn-userfeedback-sidebar">
-    <h2 data-ng-show="rating.ratingCount > 0 && loaded">
+<section class="gn-userfeedback-sidebar">
+  <h2>
```

Comment:
I noticed that the modals/popups are now nested within the `<h2>` tag which changes the fonts in the modal itself, all the text gets the styling of the `<h2>` element

Reply:
hum, I think I fixed this. Will double check.

Reply:
I saw it once this morning. But now looks ok here

![image](https://user-images.githubusercontent.com/1701393/108863554-9c9c7700-75f1-11eb-96ca-19811504c284.png)

Reply:
For me it's not solved, did you push the changes? Sorry for asking

Reply:
No i just tested again locally and it was fine. Which browser ? in debug or not you get the same ?

Reply:
It's in Chrome and it happens in both modes.

This is what my code looks like, where `data-gn-userfeedbackfull` and  `data-gn-userfeedbacknew` are within the `<h2>`:
```
  <h2>
    <i class="fa fa-comment-o"></i>
    <!--<span data-ng-show="rating.ratingCount > 0 && loaded"
            data-translate="">userFeedback</span>-->
    <span data-ng-show="rating.ratingCount > 0 && loaded">
      <span data-translate="">GUFrate</span>
      <span>{{rating.ratingAverages['-1']}} / 5 <i class="fa fa-fw fa-star"></i></span>
      <span data-ng-show="rating.ratingAverages && loaded"
         data-translate=""
         data-translate-values="{ratingCount:'{{rating.ratingCount}}'}">GUFfromRatingCountRatings</span>
    </span>
    <span data-ng-show="rating.ratingCount === 0 && loaded"
        data-translate="">GUFnoRatings</span>

    <div class="btn-toolbar pull-right">
      <div class="pull-right"
           data-gn-userfeedbackfull='record'
           data-ng-if="rating.ratingCount > 0 && record.draft != 'y'">
      </div>
      <div class="pull-right"
           data-gn-userfeedbacknew='record'
           data-ng-if="record.draft != 'y'">
      </div>
    </div>
  </h2>
```

Reply:
Still no luck on reproducing but switch to a flex approach - should be ok now ?